src/options.js

```diff
@@ -0,0 +1,10 @@
+const extend = require('lodash/extend');
+
+const DefaultSettings = {
+  libraries: ['clsx', 'classnames'],
+  functionNames: [],
+};
+
+module.exports = options => {
+  return extend({}, DefaultSettings, options);
```

Comment:
Do we really need lodash here instead of native `Object.assign`?

Reply:
Not at all, changed it to use spread operator